src/components/Filter/Filter.css

```diff
@@ -44,13 +44,12 @@
   }
 }
 
-@media all and (min-width: 576px) {
+@media all and (min-width: 800px) {
   .toggle-filter {
     display: none;
   }
   .sidebar {
-    order: 2;
-    position: fixed;
+    max-width: 265px;
```

Comment:
Same here. Would be great to have written why the `265px`

Reply:
👍